### PR TITLE
Update email-templates and other deps to latest.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
Resolves the "missing 'cheerio' package" error.

### CODE REVIEW

**Review by JoshDunnDev — APPROVED** [review]

**Review by dlongley — APPROVED** [review]
Rubberstamp. No idea if these dep upgrades are ok, relying on you/others to test in apps.

**Review by mattcollier — APPROVED** [review]
Needs a changelog entry.  Please include changelog on all PRs.

File: package.json [hunk 1]
```diff
@@ -27,18 +27,18 @@
...
-    "ejs": "^3.0.1",
-    "email-templates": "^7.0.4",
-    "nodemailer": "^6.4.3"
+    "ejs": "^3.1.6",
+    "email-templates": "^8.0.4",
+    "nodemailer": "^6.6.0"
   },
   "peerDependencies": {
-    "bedrock": "1.6.0 - 3.x"
+    "bedrock": "4.x"
```
Comment (davidlehn) [inline package.json]: Use the "^" syntax when not using ranges.  I'm guessing 4.0.0 is ok, but adjust upward if needed. 
```suggestion
    "bedrock": "^4.0.0"
```

**Comment by dmitrizagidulin** [comment]
@mattcollier added changelog entry, thanks.

File: package.json (hunk 1 above)
Comment (dmitrizagidulin) [reply package.json]: Done, thanks.

**Pull request merged**